Chat: Add File Attachments demo description (#31846)
DevExpress/DevExtreme: apps/demos/Demos/Chat/FileAttachments/description.md

> DevExtreme Chat supports file attachments. When this feature is activated ([fileUploaderOptions](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#fileUploaderOptions).**uploadFile** is defined), an “Attach” button appears in the message input field, allowing users to add files to their messages. 

Reviewer: ```suggestion
DevExtreme Chat supports file attachments. When this feature is activated ([fileUploaderOptions](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#fileUploaderOptions).**uploadFile** is specified), an “Attach” button appears in the message input field, allowing users to add files to their messages. 
```

DevExtreme Chat supports file attachments. When this feature is activated, an “Attach” button appears in the message input field, allowing users to add files to their messages. 

> To enable file attachments in Chat, specify [fileUploaderOptions](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#fileUploaderOptions).**uploadFile**.

Reviewer: Please update so that this is technically correct. I tried to shorten the intro.

```suggestion
DevExtreme Chat supports file attachments. When this feature is activated ([fileUploaderOptions](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#fileUploaderOptions).**uploadFile** is `true`), an “Attach” button appears in the message input field, allowing users to add files to their messages. 
```

When users attach files, each file is displayed in the input area with a file-type icon, basic details (name and size), upload status, and an option to remove files before sending.
<!--split-->

You can further customize the file upload process with [fileUploaderOptions](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#fileUploaderOptions) properties:

- **maxFileSize**    

> Specify the maximum allowed file size.

Reviewer: In Blazor we usually use third person in such cases: "specifies the maximum..."

I don't mind your version as long as it is consistent across your product. Just checking.

- **minFileSize**    
Specify the minimum allowed file size.
- **multiple**    
Set to `false` to limit uploads to a single file.
- **allowedFileExtensions**    
Restrict accepted file types.

> For the complete list of configuration options, refer to the [fileUploaderOptions API section](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#fileUploaderOptions).

Reviewer: ```suggestion
For the complete list of configuration options, refer to the following API section: [fileUploaderOptions](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#fileUploaderOptions).
```

> The [Attachment](/Documentation/ApiReference/UI_Components/dxChat/Types/Attachment/) type includes `name` and `size` fields. To add custom fields (such as `url`), handle the [onMessageEntered](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#onMessageEntered) event and update the message object’s [attachments](/Documentation/ApiReference/UI_Components/dxChat/Types/TextMessage/#attachments) array as needed. You can use this handler to save files to your server.

Reviewer 2: such as `url` in this demo OR in this demo it is the `url` field

> DevExtreme Chat supports file attachments. To enable this feature, specify [fileUploaderOptions](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#fileUploaderOptions).**uploadFile** - an "Attach" button appears in the input field, and users can send messages with files.

Reviewer 2: I prefer to make the demo description little bit less strict technical. For example (based on WN)

DevExtreme Chat now supports file attachments.
If file attachments feature is  activated, chat adds an “Attach” button to the message input. Uploaded files are previewed in the message box with basic details, upload status, and a file-type icon. Users can review attachments and remove unnecessary files before sending. After the message is sent, attachments appear within message bubbles. Each attachment displays a Download button 

To enable this feature...
To configure file upload behavior, use API options ... including the following:
- maxFileSize — Maximum file size.
- minFileSize — Minimum file size.
- multiple — Set to false to limit uploads to a single file.
- allowedFileExtensions — Restricts accepted file types.

Attachment metadata (name, size) is passed to onMessageEntered(link) along with the message data. Use this handler to implement file storing in your data source. You can also include custom fields in the message payload .... 

You can implement the onAttachmentDownloadClick(link) handler, allowing you to define your download logic.

OR 
DevExtreme Chat now supports file attachments.
If file attachments feature is  activated, chat adds an “Attach” button to the message input. 

To enable this feature...
To configure file upload behavior, use API options ... including the following:
- maxFileSize — Maximum file size.
- minFileSize — Minimum file size.
- multiple — Set to false to limit uploads to a single file.
- allowedFileExtensions — Restricts accepted file types.

Uploaded files are previewed in the message box with basic details, upload status, and a file-type icon. Users can review attachments and remove unnecessary files before sending. Attachment metadata (name, size) is passed to onMessageEntered(link) along with the message data. Use this handler to implement file storing in your data source. You can also include custom fields in the message payload ....

After the message is sent, attachments appear within message bubbles. Each attachment displays a Download button  You can implement the onAttachmentDownloadClick(link) handler, allowing you to define your download logic.
